src/types.ts

```diff
@@ -97,8 +97,6 @@ export type JsonApiError<ErrorCodeT = ErrorCode> = {
   detail: string;
 };
 
-export type Application = {};
-
 export type Links = {};
 
 export type Meta = {};
```

Comment:
@Exelord We can specify a stricter typing here:

```suggestion
export type Meta = { [key: string]: string | number | boolean | Meta };
```

Reply:
We can cover that later when we gonna work on Meta object :) Right now it's just to not forget about it.